### ISSUE TITLE
Poor Color Contrast - Main Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OSHA-Heat-Safety-iOS#3]
The color contrast on the default screen has gray text on a light gray background, which makes it difficult for people to read. There are other color contrast issues with the small text on the red background, for example, the "Feels like XX F" text. The text should have better color contrast and be larger so you can see it. 

WCAG 2.0 - 1.4.3 - Contrast (Minimum) - Level AA

**Comment by skeeJay** [comment]
.@jfc3-dol I'd be curious to hear what the required standard is here as well. Because the app's background colors change and are so bright as part of the design, white is really the only logical text color on top of it. I've used things like font size and opacity to distinguish between non-interactive labels and interactive buttons, since iOS 7+ doesn't use button outlines. So I'd be curious to find out what opacity might make sure we're complying with this issue......hopefully without breaking the visual simplicity of the design.

**Comment by skeeJay** [comment]
ad387df745739dea341a124d4d1710c91ebe11b5 increases the font size of the labels in the center of the screen to 17 pt, which is the default iOS label size, and increases the opacity of all non-interactive elements as high as is possible without becoming too close to 100% (the opacity of interactive buttons), as discussed.

**Comment by jfc3-dol** [comment]
These changes made it easier to read the text on the pages. Even though the color contrast could use some tweaking I'm going to close this issue.
